app/src/interfaces/useSurveyApi.interface.ts

```diff
@@ -170,6 +170,7 @@ export interface IGetSurveyAttachment {
   lastModified: string;
   size: number;
   securityToken: any;
+  securityReviewTimestamp: string;
```

Comment:
make security parameter

Reply:
Might need to elaborate on what you mean.

I think this looks correct, unless I'm forgetting something. The database column this is mapping to is `security_review_timestamp`

Reply:
Oh, unless you mean putting the security stuff in its own object like we were talking about? That definitely makes sense when it comes to the list of all security rules applied to an attachment, but this particular column is part of the attachment table, so it makes sense to live in this object.

I think ultimately, we still want to maintain a separate object that represents a single attachment record, and a separate object that represents the rules applied to a single attachment. That way we can fetch them separately via services, etc. And then our attachment related endpoints can mix and match the service calls to return attachment(s) with or without security rules alongside them.